masschallenge/impact-api
# Ac-5185 - allow impact-api to deploy a release tag

Found the golden tip here which basically suggests asking the remote repo for tag information (since travis does some odd things with the local origin):
https://github.com/travis-ci/travis-ci/issues/4745

To test this 

- make a branch off of this branch (Ac-5185b)
- make a small change and push it
- observe build and see that an image with the name of your branch is tagged in ECR (https://console.aws.amazon.com/ecs/home?region=us-east-1#/repositories/impact-api#images;tagStatus=ALL)

- publish a new test release off of your branch - give it "test tag" of some sort (instead of the normal release tag format ex: 2017.12.12)
- allow travis to run the build and on completion observe that an image with the name of the release tag (not the branch) shows up in ECR (https://console.aws.amazon.com/ecs/home?region=us-east-1#/repositories/impact-api#images;tagStatus=ALL)
